cmd/flarectl/flarectl.go

```diff
@@ -378,7 +427,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "flarectl"
 	app.Usage = "CloudFlare CLI"
-	app.Version = "2015.12.0"
+	app.Version = "2016.4.0"
```

Comment:
Yeah, that's true. I wasn't sure how to handle it for the library, other than using git tags. I'm open to ideas.